sensor/common/compliance/service_impl.go

```diff
@@ -96,7 +96,11 @@ func (s *serviceImpl) startSendingLoop() {
 				}
 			})
 		} else {
-			con := s.connectionManager.connectionMap[msg.Hostname]
+			con, ok := s.connectionManager.connectionMap[msg.Hostname]
+			if !ok {
+				log.Errorf("unable to find connection to compliance: %q", msg.Hostname)
```

Comment:
I think these logs should start with upper case
```suggestion
				log.Errorf("Unable to find connection to compliance: %q", msg.Hostname)
```

Reply:
All error logs should start with non-capitalized letter. Why do you think we should change it here? 

I see we still mix styles, but I think there was a govet rule to check this.
![Screenshot 2023-05-23 at 11 46 32](https://github.com/stackrox/stackrox/assets/114479/49041954-37a2-471a-b808-1833dce029c9)

Reply:
I tried to search for some "literature" to support this: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
> I tried to search for some "literature" to support this: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

`fmt.Errorf` and `errors.New` should definitely be lower cased, because error messages can be wrapped, and the formatting will be weird since upper case letters will appear in the middle of the error message. 

However, my comment is regarding **logging** messages. FWIW, the link you pasted states this point exactly:

> This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.

I believe these examples in your screenshot are the ones not following the Go pattern. I wonder why "unable" was often capitalized (although I see some exceptions). My code search looks a bit different:

![image](https://github.com/stackrox/stackrox/assets/6811076/4d8d5621-7d12-4c94-ae66-58612b080b6d)

Reply:
Ok, you got point :) Let me change it

Reply:
I hope, I covered most of the affected code in [31b4a11](https://github.com/stackrox/stackrox/pull/5999/commits/31b4a11525ec6adc163f020e748fbefa02b0746b)